frontend-student/src/components/coursePage/CourseSidebar.jsx

```
@@ -0,0 +1,162 @@
+                </span>
+                {milestone.title}
+              </span>
+              <svg
+                className={`w-4 h-4 transition-transform duration-300 text-blue-500 ${
+                  openMilestones[milestone.id] ? 'transform rotate-180' : ''
+                }`}
+                fill='none'
+                stroke='currentColor'
+                viewBox='0 0 24 24'
+                xmlns='http://www.w3.org/2000/svg'
+              >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)